config/types/etcd.go

```diff
@@ -32,7 +33,9 @@ var (
 )
 
 // Options can be the options for any Etcd version
-type Options interface{}
+type Options interface {
```

Comment:
This is necessary for the reflection logic to properly check this field for a `Validate` function. Without this the added validation logic to check for the schema prefixes is never called.